scripts/sql/289_orca_security_scan.up.sql

```diff
@@ -0,0 +1,52 @@
+INSERT INTO "plugin_parent_metadata" ("id", "name","identifier", "description","type","icon","deleted", "created_on", "created_by", "updated_on", "updated_by")
+VALUES (nextval('id_seq_plugin_parent_metadata'), 'ORCA SECURITY SCAN','security-image-scan','Orca Cloud Security plugin for scanning container images and detecting vulnerabilities','PRESET','https://raw.githubusercontent.com/devtron-labs/devtron/main/assets/orca_security_scan_logo.png','f', 'now()', 1, 'now()', 1);
```

Comment:
keep identifier something similar to Orca Security Scan - should include ORCA

Reply:
okay